mvlearn/decomposition/grouppca.py

```diff
@@ -304,24 +338,48 @@ def inverse_transform(self, X_transformed):
         X_transformed : list of array-likes or numpy.ndarray
             The dataset corresponding to transformed data
 
+        index: None, or int or array-like
+            int or list of ints specifying the indices of the
+            inputted views relative to the fitted views.
+            If None, there should be as many inputted views as fitted views.
+            Note that the index parameter is not available in all methods of
+            mvlearn yet.
+
         Returns
         -------
         Xs : list of arrays
             The recovered individual datasets
         """
         check_is_fitted(self)
+        if index is None:
+            index_ = np.arange(len(self.individual_projections_))
+        else:
+            index_ = np.copy(index)
+
         if self.multiview_output:
+            assert len(X_transformed) == len(index_)
             X_transformed = check_Xs(X_transformed)
             return [
                 np.dot(X, A.T) + mean
                 for X, A, mean in (
                     zip(
                         X_transformed,
-                        self.individual_embeddings_,
-                        self.individual_mean_,
+                        [self.individual_embeddings_[i] for i in index_],
+                        [self.individual_mean_[i] for i in index_],
+                    )
+                )
+            ]
+        if index is not None:
```

Comment:
Same comment as above in terms of repeated code.

Reply:
The situation is a bit different here as we do not iterate over X_transformed